app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if !logged_in? then
+     redirect '/login'
+   elsif logged_in?
+    @tweets = Tweet.all
+    erb :'tweets/tweets'
+  end
+end
+
+  get '/tweets/new' do
+    if logged_in? then
+  erb :"tweets/create_tweet"
+else redirect '/login'
+end
+end
+
+  post '/tweets' do
+    if params[:content] == "" then
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.